src/quote/getQuoteAndBuyCoverInputs.test.ts

```diff
@@ -30,6 +37,7 @@ const coverRouterCapacityResponse: CoverRouterProductCapacityResponse = {
 describe('getQuoteAndBuyCoverInputs', () => {
   let buyerAddress: Address;
   const DEFAULT_NEXUS_API_URL = 'https://api.nexusmutual.io/v2';
+  const TEST_API_URL = 'https://api.test.io/v2';
```

Comment:
adding `TEST_API_URL` as precaution so we do not accidentally call prod API if someone make changes in these tests